rfcs/text/0000-Managed-Access-Data.md

```diff
@@ -0,0 +1,346 @@
+    > bundles and collections where the user or a group that the user
+    > belongs to has been granted explicit access.
+
+-   Implicit rights: Users can access public data where they have not
+    > been granted any specific access rights. This set of tests would
+    > include checking access to files, bundles and collections.
+
+Negative tests:
+
+-   Explicit Denial: Users will be denied access to controlled data
+    > where they have not been granted explicit access. This set of
+    > tests would include checking access to files, bundles and
+    > collections for a user without access, or a user that does not
+    > belong to a group with access.
+
+**Access controls for NIST 800-53** are very well prescribed. They can
+be summarized as follows: (Consult the document for the authoritative
+definition of work)
+
+-   Privileged users are managed from a central place
+
+-   There is a prescribed onboarding and offboarding process for all
+    > users
+
+-   There is an audit trail for onboarding and offboarding
+
+-   Every "layer" of the system authenticates. Each component in the
+    > system authenticate and all the connections authenticate somehow
+    > even if on a "trusted" network.
+
+-   Operate on a least privilege model with developers and users getting
+    > the rights that they require, and not more. We will have a process
+    > for elevating privileges
+
+-   We will require a warning banner that you're entering a secure area
+    > on login and/or registration. We will want to wrap this in some
+    > sort of Terms Of Service that prompts users each time they change.
+    > Audit the acceptance of the terms of service.
+
+-   Must provide a logon and logoff capability from the UI. Ideally we
+    > will have an automatic logout due to inactivity like a bank does.
+
+This list is meant to get the flavor of what will be required by the
+regulations for NIST 800-53. The Access Control control family of checks
+exist inside a much larger framework of controls made up by the rest of
+NIST 800-53, and it should be noted that completing the other controls
+in the document would be required for certification of the DCP to FISMA
+moderate levels so that it can store controlled access data generated by
+funds from NIH grants.
+
+It should be noted again that though these controls are specific to
+access control requirements in the USA, they are expected to represent a
+good starting point for other controls from other nations.
+
+### **Unresolved Questions**
+
+### Design questions which should be resolved during the RFC process
+
+-   Are search results filtered by access control?
+
+-   (related) Is Metadata always searchable
+
+-   What certifications are required to meet the feature request
+    > "support controlled data in the DCP"? FISMA moderate? GDPR? What
+    > else?
+
+-   Should we have the researchers run the DAC? Perhaps providing them
+    > with a template?
+
```

Comment:
The DAC is run by the people who control access to the data, not by the DCP.  For NIH funded data, they provide mechanisms to help enable the DAC, but ultimately the DAC is a committee that is part of the research project.

If HCA is going to take in controlled-access data, the DCP may very well have to apply to the DAC for access.  It all on if the project's data user rules allow submitting to a third party to do analysis on their behalf.